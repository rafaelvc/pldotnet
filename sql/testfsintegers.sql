CREATE OR REPLACE FUNCTION sum2Integer(a integer, b integer) RETURNS integer AS $$
a+b
$$ LANGUAGE plfsharp;
SELECT sum2Integer(1000, 1) = integer '1001';

