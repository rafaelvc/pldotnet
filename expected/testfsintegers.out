CREATE OR REPLACE FUNCTION sum2Integer(a integer, b integer) RETURNS integer AS $$
a+b
$$ LANGUAGE plfsharp;
SELECT sum2Integer(1000, 1) = integer '1001';
 ?column? 
----------
 t
(1 row)

